include/GafferOSL/OSLShader.h

```diff
@@ -73,35 +73,6 @@ class GAFFEROSL_API OSLShader : public GafferScene::Shader
 		/// Returns an OSL metadata item from the specified shader parameter.
 		const IECore::Data *parameterMetadata( const Gaffer::Plug *plug, const IECore::InternedString &name ) const;
 
-		/// TODO - decide what to call this and where it should live
```

Comment:
This is a breaking change, so we should just deprecate the function here and make a separate PR to remove it on `main` (I'm assuming this PR should be targeting `0.61_maintenance` since that is where we plan to release USDShade support).

Reply:
Conclusion from chat just now : this PR will stay on `main`. I had mistakenly conflated it with the USDShade support, but it's not necessary for that - it's just subsequent tidying. We just need to update Changes.md to mention the removed function in the Breaking Changes section.